pkg/ops/rawDiskGeneration.go

```diff
@@ -853,7 +853,7 @@ func (r *RawImage) installGrubToDisk(image string) error {
 	// Get only the loop device without the /dev/ prefix
 	cleanLoopDevice := string(loopDevice)[5:]
 	recoveryLoop := fmt.Sprintf("/dev/mapper/%s%s", cleanLoopDevice, "p3")
-	err = unix.Mount(recoveryLoop, tmpDirRecovery, "ext2", 0, "")
+	err = unix.Mount(recoveryLoop, tmpDirRecovery, "ext4", 0, "")
```

Comment:
Why not?

```suggestion
	err = unix.Mount(recoveryLoop, tmpDirRecovery, agentConstants.LinuxFs, 0, "")
```

Reply:
The resolution is, i clicked the wrong one loool